packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -141,7 +157,7 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
             <div style={this._getStickyPlaceholderHeight(isStickyBottom)} />
           </div>
         )}
-        <div style={this._getNonStickyPlaceholderHeight()} ref={this._root}>
+        <div style={this._getNonStickyPlaceholderHeight()} ref={this._placeHolder}>
```

Comment:
The nonStickyContent needs to be appended to the placeholder now. Please fix resetSticky.

Reply:
To clarify, test the following scenario -
* Scroll down such that a component gets into sticky state
* Scroll back such that component gets non-sticky again

Check the DOM. nonStickyContent would be attached to root, instead of placeholder.

Reply:
Thanks, good catch. Fixed.